<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerControl</class>
 <widget class="QWidget" name="PlayerControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="1" column="0" colspan="3">
      <widget class="QSlider" name="seekSlider">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="PMDButton" name="rewindButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="PMDButton" name="forwardButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0" rowspan="1" colspan="1">
      <widget class="QSpinBox" name="rangeStart">
       <property name="Visible">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
         <string notr="true">
           background-color: rgb(0, 0, 0);
           color: rgb(255, 255, 255);font: 10pt;
         </string>
       </property>
       <property name="alignment">
         <set>Qt::AlignRight</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2" rowspan="1" colspan="1">
      <widget class="QSpinBox" name="rangeEnd">
       <property name="Visible">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
         <string notr="true">
           background-color: rgb(0, 0, 0);
           color: rgb(255, 255, 255);font: 10pt;
         </string>
       </property>
       <property name="alignment">
         <set>Qt::AlignRight</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="PMDButton" name="playStopButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="frameCounter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);font: 10pt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PMDButton</class>
   <extends>QPushButton</extends>
   <header>PMDButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
